dotcom-rendering/src/components/Nav/ExpandedMenu/Column.tsx

```diff
@@ -200,14 +200,30 @@ const columnStyle = css`
 	}
 `;
 
+const columnStyleWithPageSkin = css`
+	${columnStyle};
+	${from.leftCol}, ${from.wide} {
+		width: 134px;
+
+		:first-of-type {
+			width: 123px;
+		}
+	}
+`;
+
+const columnStyleBasedOnHasPageSkin = (hasPageSkin?: boolean) =>
+	hasPageSkin ? columnStyleWithPageSkin : columnStyle;
```

Comment:
Not a blocker: we tend to do these checks inline in the `css` prop

Reply:
It took me a bit to understand what you mean but thanks to @DanielCliftonGuardian comment it was able to get it. Addressed in aa0367810f0f3dada125612cb1c53b228f3bd821